features/nanostack/mbed-mesh-api/mbed_lib.json

```diff
@@ -6,6 +6,12 @@
             "help": "Nanostack's heap size [bytes: 0-‭4294967295‬]",
             "value": 32500
         },
+        "mac-neigh-table-size": {
+            "help": "SW MAC neighbour table size configure which is allocated when MAC is allocated",
```

Comment:
-Would it be enough to say "Number of devices stored to the SW MAC neighbour table"

Reply:
ye simple is beautiful

Reply:
Fixed